templates/components/modal/welcome.html

```diff
@@ -15,7 +15,7 @@
           >Welcome to the web interface of your</span
         >
         <br />
-        <span class="txt-bold">{{env.title}}</span>
+        <span class="txt-bold" data-label="msg.modal-p3">ORIGINAL PRUSA 3D PRINTER</span>
```

Comment:
why hardcode it instead of using a variable?

Reply:
Other texts were also hardcoded so I sticked to the style it was written. Texts are rewritten by translations anyway so there is no need for any placeholder